plutus-scb/src/Cardano/SigningProcess/Server.hs

```diff
@@ -59,12 +62,14 @@ app stateVar =
         (processSigningProcessEffects stateVar)
         (uncurry WE.addSignatures)
 
-main :: (MonadIO m) => SigningProcessConfig -> m ()
-main SigningProcessConfig{spWallet, spBaseUrl} = runStdoutLoggingT $ do
+main :: ServiceAvailability IO s => (MonadIO m) => SigningProcessConfig -> s -> m ()
```

Comment:
Is there any reason for having two implications instead of a tuple? (Other than that it is possible :) )

Reply:
Ah, this is a PureScript habit sneaking in. (PS _insists_ you do it this way.)

I'll amend it to the Haskell style. Thanks. :-)